connectors/google/google-drive/src/main/java/io/camunda/connector/gdrive/GoogleDriveFunction.java

```diff
@@ -49,6 +50,7 @@ public Object execute(final OutboundConnectorContext context) {
   }
 
   private GoogleDriveRequest parseVariablesToRequest(final String requestAsJson) {
+    GoogleDriveRequest googleDriveRequest = new Gson().fromJson(requestAsJson, GoogleDriveRequest.class);
```

Comment:
just add as an example how parsing will be

Reply:
I am wondering if [this one](https://camunda.slack.com/archives/C0350959WG1/p1685552122703619) is applicable. Let's have this discussion tmr on daily meeting.